app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    arr = []
```

Comment:
Could you choose another name for this variable, pls. It's unclear what is stored in it.